src/main/java/net/sf/jabref/gui/entryeditor/PdfCommentsTab.java

```diff
@@ -66,8 +66,9 @@ private void setUpPdfCommentsTab() {
             DefaultListModel<String> listModel = new DefaultListModel<>();
             commentList.setModel(listModel);
             PdfCommentImporter commentImporter = new PdfCommentImporter();
-            HashMap<String, String> importedNotes = commentImporter.importNotes(field.get());
-            for (String note : importedNotes.values()) listModel.addElement(note);
+            String pdfPath = field.get().replaceFirst(".*?:", System.getProperty("file.separator")).replaceAll(":PDF", "");
```

Comment:
Thank you for that! Is it a valid use case that more than one pdf file is linked? If so, I would have to take care about parsing multiple files and we need a UI that is capable of representing that.